inaugurator/main.py

```diff
@@ -44,6 +45,8 @@
 parser.add_argument("--inauguratorDontReadSmartData", action="store_true", default=False)
 parser.add_argument("--inauguratorDontFailOnFailedDisk", action="store_true", default=False)
 parser.add_argument("--inauguratorCleanupUpperPercentageThreshold", type=int, default=65)
+parser.add_argument("--inauguratorWipeOldInauguratorInstallations", type=bool, action="store_true",
```

Comment:
anyhow we'll have to restart rackattack. And I rather restart rackattack than by accident erase another customer